testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/sse/SseTest.java

```diff
@@ -195,6 +197,10 @@ public void testBroadcast() throws Exception
       SseEventSource eventSource2 = SseEventSource.target(target2).build();
       eventSource2.register(checkConsumer);
       eventSource2.open();
+      
+      //Let's wait for all SseEventSource to be connected
+      TimeUnit.SECONDS.sleep(3);

```

Comment:
Without this sleep time, invocations of `client.target(generateURL("/service/server-sent-events/broadcast")).request()
.post(Entity.entity(textMessage, MediaType.SERVER_SENT_EVENTS));` will try to broadcast message too early since no SseEventSINK is connected yet.